modules/ROOT/pages/6.0-release-notes-overview.adoc

```diff
@@ -0,0 +1,31 @@
+= TinyMCE 6.0 release overview
+:navtitle: Overview
+:description: TinyMCE 6.0 Release notes overview
+:keywords: releasenotes, new, changes, bugfixes, breaking changes
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+== Overview
+
+{productname} 6.0 was released for {enterpriseversion} and {cloudname} on <<`+WEEKDAY+`, `+MMM+` `+D+``+st|nd|th+`, `+YYYY+`>>. It includes {productname} 6.0 and additional changes to premium plugins. These release notes provide an overview of the changes for {productname} 6.0, including:
```

Comment:
Need to update the date here. It should now be tomorrow's date I believe. @lnewson ?

Reply:
Yeah that sounds right.

Reply:
```suggestion
{productname} 6.0 was released for {enterpriseversion} and {cloudname} on Thursday, April 6th, 2022. It includes {productname} 6.0 and additional changes to premium plugins. These release notes provide an overview of the changes for {productname} 6.0, including:
```
I'm not sure how to make the "th" superscript though sorry.

Reply:
```suggestion
{productname} 6.0 was released for {enterpriseversion} and {cloudname} on Thursday, April 7^th^, 2022. It includes {productname} 6.0 and additional changes to premium plugins. These release notes provide an overview of the changes for {productname} 6.0, including:
```
This just fixes the date and adds the markup to make `th` superscript.